models/comment-models.js

```diff
@@ -0,0 +1,18 @@
+const db = require("../db/connection");
+
+exports.selectCommentsByArticleId = (article_id) => {
+  return db.query(`
+    SELECT * FROM comments
+    WHERE article_id = $1
+    ORDER BY created_at DESC;
+    `, [article_id])
+    .then(({ rows }) => {
+      if (!rows.length) {
+        return Promise.reject({
+          status: 404,
+          message: "No comments found"
+        });
```

Comment:
This isn't expected functionality.
If there is no comments we still want to return the empty array to the user it isn't an Error.